Makefile

```diff
@@ -31,26 +24,28 @@ CRD_JSONNET_FILES += jsonnet/prometheus-operator/prometheusrule-crd.libsonnet
 CRD_JSONNET_FILES += jsonnet/prometheus-operator/servicemonitor-crd.libsonnet
 CRD_JSONNET_FILES += jsonnet/prometheus-operator/thanosruler-crd.libsonnet
 
+TOOLS_BIN_DIR ?= $(shell pwd)/tmp/bin
```

Comment:
Nice idea!  I will be happy to not worry about failures due to different versions of the tools installed to go/bin.